pkg/api/config.go

```diff
@@ -11,6 +11,8 @@ import (
 type Config struct {
 	// ClusterVersion defines release version of the plugin used to build the cluster
 	ClusterVersion string `json:"clusterVersion,omitempty"`
+	// ClusterLogLevel specifies the OpenShift component log level
+	ClusterLogLevel uint8 `json:"clusterLogLevel,omitempty"`
```

Comment:
Would it be better to make this a const?  Similar to https://github.com/openshift/openshift-azure/blob/master/pkg/api/types.go#L71

Reply:
good idea